tests/source/doc-attrib.rs

```diff
@@ -47,7 +47,6 @@ pub fn foo() {}
 
 // path attrs
 #[clippy::bar]
-#[clippy::bar=foo]
```

Comment:
why is this removed ?

Reply:
Because now this kind of attribute is considered as an invalid syntax. If we keep this the test just fails during parsing it :stuck_out_tongue_closed_eyes:.

For more details see https://github.com/rust-lang/rust/pull/57367 for the rustc's PR that changed the behavior.